examples/datasource-basic/package.json

```diff
@@ -47,10 +60,9 @@
   },
   "dependencies": {
     "@emotion/css": "^11.1.3",
-    "@grafana/data": "8.3.6",
-    "@grafana/runtime": "8.3.6",
-    "@grafana/toolkit": "8.3.6",
-    "@grafana/ui": "8.3.6",
+    "@grafana/data": "latest",
```

Comment:
I think it would be best if we stick to "pinned version numbers" as before rather than the latest keyword. In this way we can see which version of the `@grafana` packages the plugin is using without needing to look in the `yarn.lock` file.

Reply:
Good point! 👍 

Fixed in [2b03ee4](https://github.com/grafana/grafana-plugin-examples/pull/67/commits/2b03ee40e150c2993a31db9bafab2abcabcedfe3).